CHANGELOG.md

```diff
@@ -14,19 +24,22 @@
 
 ### Minor Changes
 
-- 7b71d4a: Change `Collection` to a default export. A deprecated named export still exists to make migration easier.
+- 7b71d4a: Change `Collection` to a default export. A deprecated named export
+  still exists to make migration easier.
 
 ## 0.4.1
 
 ### Patch Changes
 
-- 5efc2e3: Ensure that `this.frame` always returns a `Frame` rather than a `Page | Frame` like it used to.
+- 5efc2e3: Ensure that `this.frame` always returns a `Frame` rather than a
+  `Page | Frame` like it used to.
 
 ## 0.4.0
 
 ### Minor Changes
 
-- 1be805b: Add `nth`, `first`, and `last` properties for nested collections similar to the same methods that are available for locators.
+- 1be805b: Add `nth`, `first`, and `last` properties for nested collections
```

Comment:
Was there just some linting change for max column widths?

Reply:
Good call, looks like Changesets will now respect prettier when formatting changelogs.  I used to have to ignore those files with Prettier.

https://github.com/Widen/lariat/pull/28